src/games/gameitem.py

```diff
@@ -218,99 +218,18 @@ def tooltip(self, data):
 
 
 class GameTooltipFormatter:
-    TIP_FORMAT = str(util.THEME.readfile("games/formatters/tool.qthtml"))
+
+    icon_path = os.path.join("chat", "countries/")
```

Comment:
Forward slash might not work on windows, I'm not sure. Using the os-specific separator at the end might be a good idea.

Reply:
it works on my windows